.github/workflows/acceptance_tests.yml

```diff
@@ -0,0 +1,64 @@
+name: Acceptance tests
+
+on:
+  pull_request:
+    types:
+    - opened
+    - edited
+    - synchronize
+    - reopened
+    - closed
+
+jobs:
+  acceptance-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build all images
+        run: |
+          docker build -t py docker/py/
+          docker build -t cuda-tf --build-arg BASE_IMAGE=py docker/cuda-tf/
+          docker build -t generic --build-arg RENKU_BASE=py  docker/generic/
+          docker build -t julia --build-arg BASE_IMAGE=py docker/julia/
+          docker build -t r --build-arg RENKU_BASE=py docker/r/
```

Comment:
we should probably make sure that the build args are consistent everywhere. Probably in another issue/PR

Reply:
one more thing - we have a `Makefile` the builds these images - should that be used here for consistency?

Reply:
I forgot about that. I will use the Makefile.

Reply:
@rokroskar I will add an issue to redo the Makefile and use it in both the workflow that publishes the images and this one. Because as it stands right now the Makefile is not used in either these tests but it is not used in the workflow that publishes either.

Reply:
see #176

Reply:
ah that is true - I think the Makefile got out of sync at one point and we stopped using it. afaik it works again, but it never made it back into the CI. We could also do away with the Makefile, not sure what is better.